.github/workflows/build_cmake.yml

```diff
@@ -26,16 +26,12 @@ jobs:
             os: windows-latest,
             build_type: "Release", cc: "gcc-9", cxx: "g++-9",
             cmake_platform: "-A x64",
-            # On msvc, the -j flag does not help build times and causes intermittent
-            # file permission errors.
-            cmake_number_of_jobs: ""
           }
         - {
             name: "Ubuntu Build", artifact: "Linux.tar.xz",
             os: ubuntu-18.04,
             build_type: "Release", cc: "gcc-9", cxx: "g++-9",
             cmake_platform: "",
-            cmake_number_of_jobs: "-j 2",
```

Comment:
Same with nilrt, ubuntu went over the edge.

The "good news" is that it's increasingly clear that the problems are with core_server and device_server including everything. We can fix that, and I have a plan for it. So, as long as we can implement that and keep future services DAQ-size-or-smaller, we can turn this back on.

Reply:
I'm pretty sure this is causing the build on #370 to fail as well, so I'll wait for this to go in first.

Reply:
I'll just separate this out and push it through